solanches/models.py

```diff
@@ -290,3 +305,21 @@ def get_categoria(produto_id):
     def to_dict(self):
         produto = vars(self).copy()
         return produto
+
+
+class BlockList:
+
+    def __init__(self, token):
+        self._id = token
+    
+    def save(self):
+        self.date = datetime.utcnow()
+        DB.block_list.insert_one(vars(self))
+        return self._id
+
+    @staticmethod
+    def contains(token):
+        query = {"_id": token}
+        token_get = DB.block_list.find_one(query)
+
+        return token_get != None
```

Comment:
Se não me engano, tu consegue o mesmo efeito fazendo assim
```suggestion
        return bool(token_get)
```